### PR TITLE
{devel}[GCCcore/11.3.0] flatbuffers v2.0.7 w/ Python 3.10.4

### PR DESCRIPTION
**Pull request by Flamefire**
(created using `eb --new-pr`)

- [x] Requires https://github.com/easybuilders/easybuild-framework/pull/4180

### CODE REVIEW

File: easybuild/easyconfigs/f/flatbuffers/flatbuffers-2.0.7-GCCcore-11.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    'download_dep_fail': True,
+    'use_pip': True,
+    'sanity_pip_check': True,
+}
+
+exts_list = [
+    ('flatbuffers', version, {
+        'sources': {
+            'download_filename': SOURCE_TAR_GZ,
+            'filename': '%(name)s-python.tar.gz',
```
Comment (boegel) [inline easybuild/easyconfigs/f/flatbuffers/flatbuffers-2.0.7-GCCcore-11.3.0.eb:44]: filename used for storing should also include version `'%(name)s-python-%(version)s.tar.gz'`

```

```

File: easybuild/easyconfigs/f/flatbuffers/flatbuffers-2.0.7-GCCcore-11.3.0.eb [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        },
+        'checksums': ['0ae7d69c5b82bf41962ca5fde9cc43033bc9501311d975fd5a25e8a7d29c1245'],
+    }),
+]
+
+modextrapaths = {'PYTHONPATH': 'lib/python%(pyshortver)s/site-packages'}
+
+sanity_check_paths = {
+    'files': ['include/flatbuffers/flatbuffers.h', 'bin/flatc', 'lib/libflatbuffers.a'],
+    'dirs': ['lib/cmake'],
```
Comment (boegel) [inline easybuild/easyconfigs/f/flatbuffers/flatbuffers-2.0.7-GCCcore-11.3.0.eb:54]: also check for `'lib/python%(pyshortver)s/site-packages'` here

File: easybuild/easyconfigs/f/flatbuffers/flatbuffers-2.0.7-GCCcore-11.3.0.eb [hunk 3]
```diff
@@ -0,0 +1,57 @@
+##
+# Author:    Robert Mijakovic <robert.mijakovic@lxp.lu>
+##
+easyblock = 'CMakeNinja'
+
+name = 'flatbuffers'
+version = '2.0.7'
+
+homepage = 'https://github.com/google/flatbuffers/'
+description = """FlatBuffers: Memory Efficient Serialization Library"""
```
Comment (boegel) [inline easybuild/easyconfigs/f/flatbuffers/flatbuffers-2.0.7-GCCcore-11.3.0.eb:10]: Maybe description should be updated to mention that this easyconfigs includes the Python bindings as well?

**Review by boegel — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/f/flatbuffers/flatbuffers-2.0.7-GCCcore-11.3.0.eb (hunk 2 above)
Comment (Flamefire) [reply easybuild/easyconfigs/f/flatbuffers/flatbuffers-2.0.7-GCCcore-11.3.0.eb:54]: I thought that gets checked by the sanity check of PythonPackage? But OK, better be explicit

File: easybuild/easyconfigs/f/flatbuffers/flatbuffers-2.0.7-GCCcore-11.3.0.eb (hunk 3 above)
Comment (Flamefire) [reply easybuild/easyconfigs/f/flatbuffers/flatbuffers-2.0.7-GCCcore-11.3.0.eb]: Done

File: easybuild/easyconfigs/f/flatbuffers/flatbuffers-2.0.7-GCCcore-11.3.0.eb (hunk 2 above)
Comment (boegel) [reply easybuild/easyconfigs/f/flatbuffers/flatbuffers-2.0.7-GCCcore-11.3.0.eb]: The default `sanity_check_paths` used by `PythonPackage` (only) includes `lib/python%(pyshortver)s/site-packages`, that's right.
But if you specify `sanity_check_paths`, you're overriding the default `sanity_check_paths`, and so you should include `lib/python%(pyshortver)s/site-packages`

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3118.skitty.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/b48fb471a7de8b3b43f6cbf3aa6aec64 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2

**Comment by boegel** [comment]
Going in, thanks @Flamefire!

**Pull request merged**